### ISSUE TITLE
ACORN empty files

### ISSUE THREAD

**Issue by ggalibert** [aodn/content#433]
> how do we deal with empty files in the catalog? I have ran QC for 2013 data and found that some of the data originally on the thredds portal are empty; for instance: 10-May-2013 07:30:00; 09-Aug-2013 23:30:00; 18-Aug-2013 04:30:00, U-V and other variables are filled with nan's. I usually tend not to generate and upload these files when no data are available. what do we want to do here?
>
> -- @scosoli

I suppose for transparency it would be better to still have these files on THREDDS with NaNs everywhere. That way a user knows that the instrument has produced a file but unfortunately there was no good data collected at all during this time period. Otherwise the user will want to contact AODN or ACORN to confirm whether: 
1. there was no data collected
2. data has been collected but not transferred to AODN

**Comment by scosoli** [comment]
good, will upload the updated FV01 version retaining these empty files for consistency. thanks for confirming that

**Comment by ggalibert** [comment]
@scosoli, we have received some empty files on Wed 7 August however they didn't get successfully processed by our ingestion pipeline:

```
IMOS_ACORN_V_20100427T140000Z_TURQ_FV00_1-hour-avg.nc
IMOS_ACORN_V_20190719T040000Z_TURQ_FV00_1-hour-avg.nc
IMOS_ACORN_V_20190719T110000Z_TURQ_FV00_1-hour-avg.nc
IMOS_ACORN_V_20190722T000000Z_TURQ_FV00_1-hour-avg.nc
```
These files don't hold any data (only NaNs) and that's fine, but there is not a single global attribute nor variable attribute (metadata). At this stage the pipeline is failing because `creation_date` is missing for example. Ideally, the template used should be the same as for files with data in them. In global attribute `comment` you could even add information about the fact that after QC there was not a single good data available for this time period.

**Comment by scosoli** [comment]
I don't understand why you would have received data from 2010 on Wed 7 aug.2019, they are not even in the processing chain any more. need to investigate that a bit more.
other more recent files may have been corrupted while generating the netcdf. need to find the error and correct that. On a side note, I had to reprocess and upload radials from LANC and vectors from TURQ as channel1 and channel3 were wired incorrectly after an upgrade of an hardware module on the receiver

**Comment by scosoli** [comment]
upload of the file IMOS_ACORN_V_20100427T140000Z_TURQ_FV00_1-hour-avg.nc was successful. I have no final answer as to why it didn't go through in the previous round

**Comment by scosoli** [comment]
I have manually checked the 201907 files you listed above and yes indeed there was something funny going on there. I ran the same procedure I run in RT and generated new files for the same dates. now files are uploaded to the portal with no problem. The only thing I changed is an internal flag in the matlab procedure that was meant to check for the .nc file existence and modified that, but I don't think that was causing the issue. 
As for the previous file, I have no answer at this moment
